.github/workflows/auto_cherry_pick.yml

```diff
@@ -20,6 +20,19 @@ jobs:
       - id: set-branch
         run: echo "previous_branch=$(if [ $GITHUB_BASE_REF  == 'master' ]; then echo $(git branch -rl 'origin/6.*.z' --format='%(refname:lstrip=-1)' | sort --version-sort | tail -n1); else echo '6.'$(($(echo $GITHUB_BASE_REF | cut -d. -f2) - 1))'.z'; fi)" >> $GITHUB_OUTPUT
 
+  find-the-last-prt-comment:
+    name: Find & Save last PRT comment for auto-cherry-pick PR's
+    runs-on: ubuntu-latest
+    outputs:
+      prt_comment: ${{steps.fc.outputs.comment-body}}
+    steps:
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          issue-number: ${{ github.event.number }}
+          body-includes: "trigger: test-robottelo"
+
```

Comment:
You need `direction: last` to find the last comment in case when there are multiple comments with `trigger: test-robottelo` and there are such PRs.

Reply:
By default its `first`, we need last.

Reply:
yeah thanks for that catch